src/martian_apart_hack_sdk/backend_clients/routers.py

```diff
@@ -251,16 +293,34 @@ def poll_training_job(
         """Poll a training job until it completes or fails.
```

Comment:
[Here](https://github.com/withmartian/martian-sdk-python/pull/19) we decided to rename `poll_training_job` to `wait_training_job`. And introduce new method called `poll_training_job`, which actually _poll_ training job once (if the user wants to poll the job once manually).
While `waiting_training_job` will _wait_ for job using `poll_training_job`.